cram_pr2/cram_pr2_pick_place_demo/src/setup.lisp

```diff
@@ -67,7 +67,7 @@
   (ccl::reset-logged-owl)
 
   ;; (setf cram-tf:*transformer* (make-instance 'cl-tf2:buffer-client))
-
+  (btr:add-objects-to-mesh-list "cram_bullet_reasoning")
```

Comment:
Actually, the meshes from BTR are already in btr:*meshes*, so we don't need to call this function for that package. Delete this line

Reply:
Also need to make sure that the meshes that create conflicts are taken good care of and no information loss happens.